a/adwaita-icon-theme/manifest.x86_64.jsonc

```diff
@@ -0,0 +1,927 @@
+				"/usr/share/icons/Adwaita/symbolic/status/weather-fog-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/status/weather-overcast-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/status/weather-severe-alert-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/status/weather-showers-scattered-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/status/weather-showers-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/status/weather-snow-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/status/weather-storm-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/status/weather-tornado-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/status/weather-windy-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/checkbox-checked-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/checkbox-mixed-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/checkbox-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/focus-legacy-systray-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/focus-top-bar-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/focus-windows-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/list-drag-handle-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/pan-down-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/pan-end-symbolic-rtl.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/pan-end-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/pan-start-symbolic-rtl.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/pan-start-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/pan-up-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/radio-checked-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/radio-mixed-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/radio-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/selection-end-symbolic-rtl.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/selection-end-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/selection-start-symbolic-rtl.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/selection-start-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/window-close-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/window-maximize-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/window-minimize-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/window-new-symbolic.svg",
+				"/usr/share/icons/Adwaita/symbolic/ui/window-restore-symbolic.svg",
+				"/usr/share/licenses/adwaita-icon-theme/COPYING",
+				"/usr/share/licenses/adwaita-icon-theme/COPYING_CCBYSA3",
+				"/usr/share/licenses/adwaita-icon-theme/COPYING_LGPL"
+			],
+			"name": "adwaita-icon-theme"
+		},
+		"adwaita-icon-theme-devel": {
+			"build-depends": [
+				"binary(gtk-update-icon-cache)"
+			],
+			"files": [
+				"/usr/share/pkgconfig/adwaita-icon-theme.pc"
+			],
+			"name": "adwaita-icon-theme-devel",
+			"provides": [
+				"pkgconfig(adwaita-icon-theme)"
+			]
```

Comment:
Should this go into the main package? Not sure if we need a separate package just for the pkgconfig